### PR TITLE
Blog August InvenioRDM release

### PR DESCRIPTION
**Pull request by saragon02**
Request review from @fenekku @lnielsen @jbenito3

### CODE REVIEW

**Comment by fenekku** [comment]
Closing in favour of rebased version: https://github.com/inveniosoftware/inveniosoftware.org/pull/137

File: content/blog/2020-09-01-august-release/contents.lr [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+---
+author: Sara Gonzales, Guillaume Viger
+---
+body:
+
+We are happy to announce InvenioRDM Alpha 10 (August release)! Thank you to our team members for their efforts on this release.
+
+### What's new?
+
+With less resources than usual, we made some limited changes this month. However, three key areas *were* tackled.
```
Comment (jbenito3) [inline content/blog/2020-09-01-august-release/contents.lr:11]: It's true that there were fewer resources in the InvenioRDM team, but there was an important number of people joining Invenio Sprint, which brought new features to RDM as well :)

**Comment by jbenito3** [comment]
LGTM @fenekku @saragon02 Thanks!

**Comment by jbenito3** [comment]
oops...just noticed I am in the wrong PR :D

**Pull request closed without merging**